mon-pix/app/templates/components/user-account-update-email.hbs

```diff
@@ -0,0 +1,53 @@
+<PixBlock>
+    <div class="user-account-update-email">
+      <h3>{{t 'pages.user-account.account-update-email.title'}}</h3>
+      <div class="user-account-update-email__informations">
+        <span>{{t 'pages.user-account.account-update-email.informations' htmlSafe=true}}</span>
+      </div>
+      <form {{on 'submit' this.saveNewEmail}} class="sign-form__body user-account-update-email__form">
+        <div class="sign-form-body__input" data-test-new-email>
+          <FormTextfield @label="{{t 'pages.user-account.account-update-email.fields.new-email.label'}}"
+                       @textfieldName="newEmail"
+                       @inputBindingValue={{this.newEmail}}
+                       @onValidate={{this.validateNewEmail}}
+                       @validationMessage={{this.newEmailValidation.message}}
+                       @validationStatus={{this.newEmailValidation.status}}
+                       @autocomplete="off"
+                       @require={{true}}/>
+        </div>
+
+        <div class="sign-form-body__input" data-test-new-email-confirmation>
+          <FormTextfield @label="{{t 'pages.user-account.account-update-email.fields.new-email-confirmation.label'}}"
+                       @textfieldName="newEmailConfirmation"
+                       @inputBindingValue={{this.newEmailConfirmation}}
+                       @onValidate={{this.validateNewEmailConfirmation}}
+                       @validationMessage={{this.newEmailConfirmationValidation.message}}
+                       @validationStatus={{this.newEmailConfirmationValidation.status}}
+                       @autocomplete="off"
+                       @require={{true}}/>
+        </div>
+        {{#if this.errorMessage}}
+        <div class="user-account-update-email__error">
+          <span data-test-error-message>{{this.errorMessage}}</span>
+        </div>
+        {{/if}}
+        <div class="user-account-update-email__actions">
+          <PixButton
+                  class="user-account-update-email-actions__button"
+                  @type="button"
+                  @triggerAction={{@disableEmailEditionMode}}
+                  @backgroundColor="transparent"
+                  data-test-cancel-email>
+          {{t 'common.actions.cancel'}}
+          </PixButton>
+          <PixButton
+                  class="user-account-update-email-actions__button"
+                  @type="submit"
+                  @triggerAction={{this.saveNewEmail}}
```

Comment:
Il y a une duplication d'appel à `this.saveNewEmail` (cf. ligne 7 : `<form {{on 'submit' this.saveNewEmail}} ...`)

Reply:
Effectivement, à la base j'avais uniquement appelé` this.saveNewEmail` via le triggerAction, après un retour de Laura sur cette PR, j'ai ajouté l'appel sur le form, et j'ai enlevé le triggerAction. 
Sauf qu'en l'enlevant j'obtiens cette erreur : `Uncaught (in promise) Error: @triggerAction params is required for PixButton !`
Je laisse donc le triggerAction et retire l'appel du form ( il faudra régler ce problème sur Pix-ui )

Reply:
Tu peux éventuellement supprimer le @type   (type=button par defaut).

Concernant Pix-ui, Il faudra poser la question aux dev : peux-t-on utiliser pix-ui (type submit) avec un <Form> ?